src/backend/api/Fusion.Resources.Api/Startup.cs

```diff
@@ -104,6 +113,32 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseEndpoints(endpoints =>
             {
                 endpoints.MapControllers();
+
+                // TODO: Remove
```

Comment:
Remove?

Reply:
The whole point is to create a layer that will produce errors so martin can test errorhandling in the api. However we do not intend to let this make it's way to production, after we are done testing this.. Therefore comments for removal..